src/performance.js

```diff
@@ -173,8 +211,15 @@ export class Performance {
    *    `tickDelta` instead.
    */
   tick(label, opt_from, opt_value) {
-    if (this.tick_) {
-      this.tick_(label, opt_from, opt_value);
+    opt_from = opt_from == undefined ? null : opt_from;
+    opt_value = opt_value == undefined ? timer.now() : opt_value;
+
+    if (this.viewer_) {
+      this.viewer_.tick(JSON.stringify({
+        label: label,
```

Comment:
Do these names need to be quoted to ensure they are not obfuscated by the compiler?